.clj-kondo/config/modules/config.edn

```diff
@@ -499,8 +500,9 @@
           metabase.lib.ident
           metabase.lib.join.util
           metabase.lib.metadata
+          metabase.lib.metadata.cached-provider
           metabase.lib.metadata.calculation
-          metabase.lib.metadata.jvm
+          metabase.lib.metadata.invocation-tracker
```

Comment:
This seems reasonable to me, especially if we're eventually going to split out `lib-metadata` into a separate module with it's own `.core` namespace, but would be interested to hear your take on why you decided to add these new API namespaces rather than re-export stuff from `.core`.

Reply:
In this PR I wasn't trying to clean up the big hairball that is the set of `lib` API namespaces; I was just updating the config to capture the current state of the world. 

Consolidating the `lib` API namespaces is a project for a later day